src/stories/charts/bullet/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { ResponsiveBullet } from "@nivo/bullet";
+import { BulletChartProps } from "./_types";
+import {
+  CHARTS_COLOR_SCHEME_MONO,
+  DEFAULT_CHARTS_THEME,
+} from "../../theme/charts";
+import {theme as unguessTheme} from "../../theme";
+import { ChartContainer } from "../ChartContainer";
+import { CustomBulletChartMarker } from "./CustomBulletChartMarker";
+import { CustomBulletChartRange } from "./CustomBulletChartRange";
+
+const BulletChart = ({ theme, height, width, lineHeight, bulletRadius, ...props }: BulletChartProps) => (
+  <ChartContainer width={width} height={height}>
+    <ResponsiveBullet
+      theme={{
+        ...DEFAULT_CHARTS_THEME,
+        ...theme,
+      }}
+      measureSize={lineHeight || .1}
+      rangeColors={CHARTS_COLOR_SCHEME_MONO[1]}
```

Comment:
Ma questi `CHARTS_COLOR_SCHEME_MONO` sparsi in giro? non mi sembra un modo leggibile di inserire colori